### ISSUE TITLE
[Python 3.8] SystemError: Python version should be 3.{5, 6, 7}

### ISSUE THREAD

**Issue by TeddyCr** [nalepae/pandarallel#92]
When initializing `pandarallel.initialize()` with `progress_bar=True` on Python 3.8+ I am getting the following error

```
> File "/home/tcrepineau/.local/lib/python3.8/site-packages/pandarallel/utils/inliner.py", line 32, in wrapper raise SystemError("Python version should be 3.{5, 6, 7}")
> SystemError: Python version should be 3.{5, 6, 7} 
```
Though, I can run Pandarallel without issue when setting `progress_bar=False`

**Comment by rosstex** [comment]
Thank you for mentioning progress_bar, you just saved me big time.

**Comment by Lovestarni** [comment]
hope author can fix this bug soon

**Comment by wiseyoungbuck** [comment]
It would be nice to have the progress bar back. Thanks for this this solved my problem as well.

**Comment by nalepae** [comment]
Fixed in `pandarallel v1.5.1`

**Comment by GDelevoye** [comment]
Just encountered the same problem with Python 3.9.5, even with pandarallel v1.5.1 or later, installed with pip in a conda virtual environment
